packages/bpk-component-modal/src/BpkModalDialog-test.js

```diff
@@ -70,6 +70,45 @@ describe('BpkModalDialog', () => {
     expect(tree).toMatchSnapshot();
   });
 
+  it('should render correctly with wide prop', () => {
+    const tree = renderer.create(
+      <BpkModalDialog
+        id="my-modal"
+        className="my-classname"
+        wide
+        title="Modal title"
+        onClose={jest.fn()}
+        closeLabel="Close"
+        closeEvents={closeEvents}
+        getDialogRef={jest.fn()}
```

Comment:
This prop is changed on master `dialogRef` instead of `getDialogRef`

Reply:
I'll rebase then 😄

Reply:
Sorry I missed the point, I'll update the test